src/plays/photoshopApp/styles.css

```diff
@@ -0,0 +1,82 @@
+/* enter stlyes here */
+*, *::before, *::after {
+    box-sizing: border-box;
+  }
+  
+  body {
+    margin: 0;
+  }
```

Comment:
Why we are doing this? this will override the other styles..

Reply:
I have been doing this to remove the default browser styles..Thanks for the suggestion i was not aware that it will override other styles...I will remove it Sir thankyou

Reply:
Done and Committed Sir